sdk/core/System.ClientModel/tests/ModelReaderWriter/Models/ModelTests.cs

```diff
@@ -71,12 +72,82 @@ protected void RoundTripTest(string format, RoundTripStrategy<T> strategy)
             var data = strategy.Write(model, options);
             string roundTrip = data.ToString();
 
-            Assert.That(roundTrip, Is.EqualTo(expectedSerializedString));
+            // we validate those are equivalent element, representing the same json object (ignoring the spaces and orders, etc)
+            AssertJsonEquivalency(expectedSerializedString, roundTrip);
```

Comment:
It is not quite flexible if we are just comparing the string results because depending on the implementation of the serialization, the same model could result in different json strings but they represent the same JSON object.